### ISSUE TITLE
license_family is not generated or stubbed out

### ISSUE THREAD

**Issue by scopatz** [marcelotrevisani/grayskull#98]
Many recipes are now being submitted to conda-forge without this field.

**Comment by marcelotrevisani** [comment]
I thought it was not that important since we have the ``license``.
But I will add it then, thanks! :)

**Comment by scopatz** [comment]
Yeah, I am having to tell everyone to manually add it. Thanks for addressing this!

**Comment by marcelotrevisani** [comment]
I was doing some research and I discovered that the concept of ``license family`` is not widely used, I can understand the motives since you have the license it can be very different from that one which was based on.
One thing which is useful is to know if a license is compatible with others, but that is not the scope of ``grayskull``

I looked on
http://oss-watch.ac.uk/apps/licdiff/
https://www.opensource.org
https://spdx.org/licenses/
https://tldrlegal.com/
and even on wikipedia
https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses

None of them use license family or something similar.

So, the problem here is that there is no reliable and pragmatic way to discover the license family. The only thing which I think I can do is add just the first name of the spdx license, but it will be just a guess and it is very error-prone.

At least for now I think I will postpone this until we find a proper way to search and get the license family based on the license spdx... :/

If you have any ideas how to address this please let me know

**Comment by scopatz** [comment]
I think you just need to have a dict that maps spdx identifiers and other common license names to the license family.  For everything else that isn't in the dict, you can use `OTHER`
